lib/instrumentation/reporting.ts

```diff
@@ -89,7 +89,7 @@ export async function exportStats(config: RenovateConfig): Promise<void> {
         ContentType: 'application/json',
       };
 
-      const client = getS3Client();
+      const client = getS3Client(config.s3Endpoint, config.s3PathStyle);
```

Comment:
Is this required because we didn't initialized the global config at that time?

Reply:
Yes, we do not have global config class initialized.

Reply:
Yes it is intialized incase everything runs fine. But incase some error occurs and renovateRepository fn does not get called the GlobalConfig.set will not be called.

Therefore, I passed the values directly from global config incase that happens, to be safe.

Reply:
https://github.com/renovatebot/renovate/blob/daf48c7f06f85e8797d9ba6ae9ae7027b22d885f/lib/workers/global/index.ts#L217

need it here